tools/make/helm.mk

```diff
@@ -32,4 +32,4 @@ helm-release: helm-generate
 	helm package charts/${CHART_NAME} --app-version ${TAG} --version ${CHART_VERSION} --destination ${OUTPUT_DIR}/charts/
 
 helm-generate:
-	IMAGE=${IMAGE} RELEASE_TAG=${TAG} go run helm/*
+	ImageRepository=${IMAGE} ImageTag=${TAG} envsubst < charts/gateway-helm/values.tmpl.yaml > ./charts/gateway-helm/values.yaml
```

Comment:
I am sure the Env Name is correct. When release Helm chart, we use IMAGE and RELEASE_TAG from release workflow. ImageRepository and ImageTag are not set in release workflow.

Reply:
@Xunzhuo oh, I see what you mean. when helm chart releasing, you'll set a `TAG` environment variable. [release](https://github.com/envoyproxy/gateway/blob/main/.github/workflows/release.yaml#L34) but it's helm tag(may be it called helm app version), not *gateway* image tag. Should we set two different variable names to differentiate this?

Reply:
Plz add changes like shown above.

Reply:
added, please check again.